### ISSUE TITLE
CMS — 508-defect-4 [AXE-CORE]: All page content MUST be contained by landmarks

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#7757]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

Team & project: vsa-public-websites, AMA follow-on

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Moderate Template Issue | All page content **must** be contained by landmarks
.vads-u-padding-x--1 = `<div class="vads-u-padding-x--1 medium-screen:vads-u-padding-x--2">`

1. After you request a decision review — https://preview-prod.vfs.va.gov/preview?nodeId=3008
1. Supplemental Claims — https://preview-prod.vfs.va.gov/preview?nodeId=3011
1. Higher-Level Reviews — https://preview-prod.vfs.va.gov/preview?nodeId=3003
1. Board Appeals — https://preview-prod.vfs.va.gov/preview?nodeId=3029
1. Contested Claims — https://preview-prod.vfs.va.gov/preview?nodeId=3030
1. Fiduciary claims — https://preview-prod.vfs.va.gov/preview?nodeId=3016
1. Insurance claims — https://preview-prod.vfs.va.gov/preview?nodeId=3025
1. FAQs — https://preview-prod.vfs.va.gov/preview?nodeId=3004
1. Get help with review request — https://preview-prod.vfs.va.gov/preview?nodeId=3009
1. Board hearings with a VLJ — https://preview-prod.vfs.va.gov/preview?nodeId=3010

### Why it matters

Screen reader users can navigate to a section based on its HTML element or ARIA Landmark. It is best practice to contain all content excepting skip links, within distinct regions such as the header, nav, main, and footer. As this content is relatively decorative, it is seems less urgent. However, since screen reader users can't tell what is and isn't decorative, containing content within landmarks as the rule describes will prevent confusion.

Navigating a web page is far simpler for screen reader users if the content splits between multiple high-level sections. Content outside of sections is difficult to find, and the content's purpose may be unclear.

Historically, HTML lacked some key semantic markers such as the ability to designate sections of the page as the header, navigation, main content, and footer. Using both HTML5 elements and ARIA landmarks in the same element is considered a best practice, but the future favors using native HTML5 element regions as browser support increases.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to navigate the content of the page by landmark regions.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: any
* Server destination:  production

## Steps to Recreate

1. Enter any of the above urls in browser, in the VA environment
2. Run axe scan using browser tools
3. Verify that the div.vads-u-padding-x--1 is identified as not contained by landmarks.

## Possible Fixes (optional)

This requires a page template change.

## WCAG or Vendor Guidance (optional)

* [axe-core 3.4 - All page content must be contained by landmarks](https://dequeuniversity.com/rules/axe/3.4/region?application=AxeChrome)

## Screenshots

![Screen Shot 2020-04-08 at 12 25 53 PM](https://user-images.githubusercontent.com/57469/78809012-46e8bd00-7994-11ea-9ed9-97c5e1386111.png)

**Comment by laflannery** [comment]
No longer relevant, it looks as though the HTML for the page has change since this was reported.  Will close in 1 week unless told otherwise
